src/backend/utils/api_utils.py

```diff
@@ -0,0 +1,43 @@
+import requests
```

Comment:
Is this a testfile used for testing?
Then move it under test/integrationsTests/utils/api_utils.py instead of src/backend/utils/api_utils.py